modules/reference/pages/rpk/rpk-debug/rpk-debug-remote-bundle-download.adoc

```diff
@@ -29,6 +29,8 @@ rpk debug remote-bundle download [flags]
 
 |-o, --output |string |The file path where the debug file will be written (default `./<timestamp>-remote-bundle.zip`).
 
+|--upload-url |string |If provided, where to upload the bundle in addition to creating a copy on disk.
```

Comment:
I think we could probably more clearly differentiate --upload-url from --output, for example should upload URL be an object storage service location like S3?